package.json

```diff
@@ -50,41 +50,40 @@
     "statements": 99
   },
   "devDependencies": {
-    "@commitlint/cli": "^19.2.2",
-    "@commitlint/config-conventional": "^19.2.2",
+    "@commitlint/cli": "^19.6.0",
+    "@commitlint/config-conventional": "^19.6.0",
     "@eslint-community/eslint-plugin-eslint-comments": "^4.3.0",
     "@eslint/eslintrc": "^3.0.2",
-    "@eslint/js": "^9.0.0",
-    "@release-it/conventional-changelog": "^8.0.1",
-    "@types/eslint": "^8.56.9",
+    "@eslint/js": "^9.16.0",
+    "@release-it/conventional-changelog": "^9.0.3",
+    "@types/eslint": "^9.6.1",
     "@types/estree": "^1.0.5",
-    "@typescript-eslint/parser": "^7.7.0",
-    "@typescript-eslint/utils": "^7.7.0",
-    "chai": "^4.4.1",
-    "dirty-chai": "^2.0.1",
```

Comment:
the package was removed: it was not used anywhere and seems no longer maintained - the last version was published ~7yrs ago.